dsaps/models.py

```diff
@@ -211,3 +211,39 @@ def elapsed_time(start_time, label):
     """Calculate elapsed time."""
     td = datetime.timedelta(seconds=time.time() - start_time)
     logger.info(f'{label} : {td}')
+
+
+def metadata_csv(row, metadata_rec, key, field, language, delimiter):
```

Comment:
It would be simpler to set a default value for delimiter, eg `delimiter=None` and then check against that below, so you don't have to pass  an empty string every time you call this function with no delimiter. Same for language.

It would also be better practice to have this function either return this result list and assign it to the original `metadata_rec` variable, or return a list of metadata elements and then append those to `metadata_rec`.